daemon/info.go

```diff
@@ -64,7 +64,7 @@ func (daemon *Daemon) SystemInfo(ctx context.Context) (*system.Info, error) {
 		OSType:             runtime.GOOS,
 		Architecture:       platform.Architecture(),
 		RegistryConfig:     doWithTrace(ctx, "registry.ServiceConfig", daemon.registryService.ServiceConfig),
-		NCPU:               doWithTrace(ctx, "sysinfo.NumCPU", sysinfo.NumCPU),
+		NCPU:               doWithTrace(ctx, "runtime.NumCPU", runtime.NumCPU),
```

Comment:
Nit; we could even consider removing the `doWithtrace` here, as it would now be tracing go stdlib; not sure if there's much use in that.